emote/callbacks.py

```diff
@@ -377,13 +378,15 @@ def __init__(
         checkpoint_index: int = 0,
         optimizers: Optional[List[optim.Optimizer]] = None,
         networks: Optional[List[nn.Module]] = None,
+        storage_subdirectory: str = "checkpoints"
```

Comment:
Should probably rename `path` to `run_root` or similar. And update the docstring above. :-)